src/config.js

```diff
@@ -116,4 +117,19 @@ function prepareLauncher(launcher, config) {
                 launcher.setenv(key, config.environment[key], true);
         });
     }
+    if (options.debug)
+        launcher.flags |= Gio.SubprocessFlags.STDIN_INHERIT;
```

Comment:
It's odd because the self-tests pass, but I get this error when running with `--debug`:

```
g_object_set_is_valid_property: construct property "flags" for object 'GSubprocessLauncher' can't be set after construction
```
Maybe `prepareLanucher()` should just return the `GSubprocessLauncher`?

Reply:
I neglected to test this one last time before making the change. It's super bizarre; `flags` is construct-only but nonetheless there is a [`set_flags()`](https://gjs-docs.gnome.org/gio20~2.64p/gio.subprocesslauncher#method-set_flags) method.

(The self-test passes because we use a mock object for the launcher.)

Reply:
Ah I see that now, it's actually `write/construct-only` (even stranger) so I'm not sure re-reading the flags after the fact will work either.